### PR TITLE
Add overflow and wordWrap options to treeMap

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Comment by alexanderPolosatov** [comment]
Hi, @RomanTsukanov 

Please, focus on names of enums.

File: js/docEnums.js [hunk 1]
```diff
@@ -70,6 +70,16 @@
  * @member 'Default' @deprecated 'Material'
  */
 
+/**
+ * @typedef {string} Enums.VizWordWrap
+ * @enum {'normal'|'break-word'|'none'}
+ */
+
+/**
+  * @typedef {string} Enums.VizOverflow
```
Comment (roman-simionov) [inline js/docEnums.js]: I suggest to rename to 'VizTextOverflow'

Comment (alexanderPolosatov) [reply js/docEnums.js]: @RomanTsukanov ?

File: js/viz/core/renderers/renderer.js [hunk 2]
```diff
@@ -983,6 +995,8 @@ function wordWrap(text, maxWidth, ellipsisMaxWidth, options) {
                 if(text.startBox + text.tspan.getSubStringLength(0, i) < ellipsisMaxWidth) {
                     setNewText(text, i, ELLIPSIS);
                     break;
+                } else {
```
Comment (roman-simionov) [inline js/viz/core/renderers/renderer.js]: it would be better using 'else if'

**Review by roman-simionov — CHANGES_REQUESTED** [review]

File: js/docEnums.js (hunk 1 above)
Comment (RomanTsukanov) [reply js/docEnums.js]: VizTextOverflow is good

File: js/viz/docs/doctreemap.js [hunk 3]
```diff
@@ -186,7 +187,19 @@ var dxTreeMap = {
...
-            }
+            },
+            /**
+            * @name dxTreeMapOptions.tile.label.wordWrap
+            * @type Enums.VizWordWrap
+            * @default "normal"
+            */
+            wordWrap: "normal",
+            /**
+            * @name dxTreeMapOptions.tile.label.overflow
```
Comment (RomanTsukanov) [inline js/viz/docs/doctreemap.js]: I know that we've decided on this name with @pavelgruba, but now it seems to me that we should prefix it with `text`: `textOverflow`. "Label overflow" sounds a bit weird.

**Comment by alexanderPolosatov** [comment]
@roman-simionov , @RomanTsukanov 
Guys, I renamed options

**Review by RomanTsukanov — APPROVED** [review]

**Review by roman-simionov — APPROVED** [review]

**Pull request merged**